traffic_ops/traffic_ops_golang/deliveryservice/servers/servers.go

```diff
@@ -724,6 +724,21 @@ func GetReadAssigned(w http.ResponseWriter, r *http.Request) {
 		api.WriteAlertsObj(w, r, http.StatusOK, alerts, v3ServerList)
 		return
 	}
+
+	// Based on version we load Delivery Service Server - for version 5 and above we use DSServerV5
```

Comment:
Since you are using the same logic in two places, can it not be combined in a function call?